lib/src/widgets/ensure_visible.dart

```diff
@@ -38,7 +42,32 @@ class EnsureVisible extends StatefulWidget {
 }
 
 class EnsureVisibleState extends State<EnsureVisible> {
-  void ensureVisible() {
+  /// If you omit [preciseAlignment] or pass `null`, the widget will just be scrolled into view,
+  /// i.e. the scroll view will be scrolled just enough for the widget to appear at the bottom
+  /// or top of the viewport.
+  /// This will take a duration specified by [duration] and follow a curve specified by [curve].
+  /// The [duration] defaults to 100 milliseconds and the [curve] to [Curves.ease].
+  ///
+  /// If you want to have the widget scrolled to a specific alignment, you need to set [preciseAlignment].
+  /// This value represents to what extent your widget should be scrolled into the viewport:
+  ///
+  ///   * `preciseAlignment: 0` will scroll your widget to the start of your viewport.
+  ///   * `preciseAlignment: 0.5` will scroll your widget to the center of your viewport.
+  ///   * `preciseAlignment: 1` will scroll your widget to the end of your viewport.
+  ///   * `preciseAlignment: null` or just omitting it will let the package handle the alignment
+  ///     and will just scroll your widget into view.
+  Future<void> ensureVisible({
+    Duration duration = const Duration(milliseconds: 100),
+    Curve curve = Curves.ease,
+    double preciseAlignment,
```

Comment:
I only wanted to add `preciseAlignment` initially, but in the process I realized that it is really counterintuitive to specify the `duration` and `curve` for the whole widget, so I also moved those to the function.